src/kernels/MIOpenReduceExtreme.cpp

```diff
@@ -70,3 +71,69 @@ extern "C" __global__ void ExtremeFwdContiguous(const INPUT_TYPE* __restrict__ x
     extremefwdcontiguous<INPUT_TYPE, OUTPUT_TYPE, OP_TYPE>(
         x, y, indice, output_numel, reduce_size, inner_size);
 }
+
+template <typename TI, typename TO>
+__device__ void aminmaxBwd(const TI* __restrict__ input,
+                           TO* input_grad,
+                           const TI* output,
+                           const TI* output_grad,
```

Comment:
you should remove `__restrict__` or use it for all parameters

Reply:
resolved